crypto/fipsmodule/ec/ecp_nistz256.c

```diff
@@ -45,6 +45,7 @@ static const Limb ONE[P256_LIMBS] = {
 /* Precomputed tables for the default generator */
 #include "ecp_nistz256_table.inl"
 
+#ifndef OPENSSL_MIPS64
```

Comment:
Instead of this condition, let's define a symbol `GFp_P256_ASM` or similar that indicates whether we have assembly code or not, and use that. There should be no mention of MIPS in this file. Instead, I would expect all the conditions to be `#if defined(GFp_P256_ASM)` or `#if !defined(GFp_P256_ASM)`